justfile

```diff
@@ -0,0 +1,18 @@
+# Like GNU `make`, but `just` rustier.
+# https://just.systems/man/en/chapter_19.html
+# run `just` from this directory to see available commands
+
+# Default command when 'just' is run without arguments
+# Run 'just <command>' to execute a command.
+default: help
+
+# Display help
+help:
+    @printf "\nRun 'just -n <command>' to print what would be executed...\n\n"
+    @just --list
+    @echo "\n...by running 'just <command>'.\n"
+    @echo "This message is printed by 'just help' and just 'just'.\n"
+
+# Lint nix files
+lint:
+    nix run nixpkgs#nixpkgs-fmt -- .
```

Comment:
Replace this with:

```
lint:
    nix fmt
```

Then add `formatter = pkgs.nixpkgs-fmt` under `perSystem` of `flake.nix`.

Reply:
https://github.com/cameronraysmith/nix-dev-home/commit/5a2bc1fedc9791adbe7ac41f596df1eb9fc4253f